zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
 // Test Case: Bad state id
 func (t *ZZKTest) TestHostStateListener_Spawn_BadStateID(c *C) {
-	// Pre-requisites
-	conn, err := zzk.GetLocalConnection("/")
-	c.Assert(err, IsNil)
-	handler := &mocks.HostStateHandler{}
 
-	// Basic set up
-	svc := &service.Service{
-		ID:     "serviceid",
-		Name:   "serviceA",
-		PoolID: "poolid",
-	}
-	spth := "/services/serviceid"
-	sdat := &ServiceNode{Service: svc}
-	err = conn.Create(spth, sdat)
-	c.Assert(err, IsNil)
-	err = conn.CreateDir("/hosts/hostid")
-	c.Assert(err, IsNil)
+	conn := setUpServiceAndHostPaths(c)
```

Comment:
The thing that comes to mind for me is the facade/service* tests.  I find that those difficult to follow, and they follow the same pattern of using those consolidated calls.